### PR TITLE
[K8S] Added - ARM information to docs

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: containers/kubernetes/how-to/deploy-x86-arm-images.mdx [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+kind: Pod
+metadata:
+  name: example-pod-with-affinity
+spec:
+  affinity:
+    nodeAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        nodeSelectorTerms:
+        - matchExpressions:
+          - key: beta.kubernetes.io/arch
```
Comment (bene2k1) [inline containers/kubernetes/how-to/deploy-x86-arm-images.mdx:60]:
```suggestion
          - key: kubernetes.io/arch
```

File: containers/kubernetes/how-to/deploy-x86-arm-images.mdx [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+<Message type="tip">
+  Using multi-arch images you benefit from a
+    - simplified deployment process.
+    - scalable solution for managing multi-architecture clusters.
+</Message>
+
+
+#### Example
+
+Below you can find an example of a pod configuration with affinity set to target the `beta.kubernetes.io/arch=arm64` label, which is present by default on Scaleway ARM nodes:
```
Comment (bene2k1) [inline containers/kubernetes/how-to/deploy-x86-arm-images.mdx:47]:
```suggestion
Below you can find an example of a pod configuration with affinity set to target the `kubernetes.io/arch=arm64` label, which is present by default on Scaleway ARM nodes:
```

File: containers/kubernetes/how-to/deploy-x86-arm-images.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+            operator: In
+            values:
+            - arm64
+  containers:
+  - name: my-container
+    image: my-image:latest
+    # Add more container configurations as needed
+```
+
+In this example, the pod's affinity is configured to be scheduled on nodes that have the label `beta.kubernetes.io/arch` with the value `arm64`.
```
Comment (bene2k1) [inline containers/kubernetes/how-to/deploy-x86-arm-images.mdx:70]:
```suggestion
In this example, the pod's affinity is configured to be scheduled on nodes that have the label `kubernetes.io/arch` with the value `arm64`.
```

File: containers/kubernetes/how-to/deploy-x86-arm-images.mdx [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+    image: my-image:latest
+    # Add more container configurations as needed
+```
+
+In this example, the pod's affinity is configured to be scheduled on nodes that have the label `kubernetes.io/arch` with the value `arm64`.
+This ensures that the pod will only be scheduled on nodes with this architecture.
+
+## Using architecture-specific deployments
+
+In cases where multi-arch images are not possible, architecture-specific deployments can be an alternative.
```
Comment (nerda-codes) [inline containers/kubernetes/how-to/deploy-x86-arm-images.mdx:75]:
```suggestion
You can also use architecture-specific deployments as an alternative, if multi-arch images are not available.
```

File: containers/kubernetes/troubleshooting/containers-not-starting-arm-instances.mdx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+# Add your application setup steps here
+```
+Additionally, ensure that any libraries or dependencies used in your application are compatible with ARM architecture.
+
+### Using multi-arch images:
+
+Use multi-architecture images that support both x86 and ARM architectures. Docker supports manifest lists, which are a collection of different image manifests for different architectures. 
+Kubernetes can pull the appropriate image based on the architecture of the node where it's scheduled to run.
+
+Example manifest list usage:
```
Comment (nerda-codes) [inline containers/kubernetes/troubleshooting/containers-not-starting-arm-instances.mdx:48]:
```suggestion
Example of manifest list usage:
```

**Comment by tgenaitay** [comment]
ARM-based nodes are now supported in beta stage. Good to go live 👍

**Review by nerda-codes — APPROVED** [review]

File: containers/kubernetes/how-to/deploy-x86-arm-images.mdx [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+1. Build multi-arch images. Docker supports multi-arch builds using `buildx`.
+2. Push the built images to a container registry accessible by your Kubernetes cluster. For example, you can use the [Scaleway Container Registry](/containers/container-registry/quickstart/).
+3. Specify node selectors and affinity. Use either [node selectors](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#nodeselector) and [affinity rules](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) to ensure pods are scheduled on nodes with compatible architectures.
+    <Message type="tip">
+      Alternatively, use taints to mark nodes with specific architectures and tolerations to allow pods to run on those nodes. Refer to the [official Kubernetes documentation](https://kubernetes.io/docs/concepts/scheduling-eviction/taint-and-toleration/) for more information regarding taints and tolerations.
+    </Message>
+#### Example
+
+Below you can find an example of a pod configuration with affinity set to target the `kubernetes.io/arch=arm64` label, which is present by default on Scaleway ARM nodes:
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/deploy-x86-arm-images.mdx:42]:
```suggestion
Below, you can find an example of a pod configuration with affinity set to target the `kubernetes.io/arch=arm64` label, which is present by default on Scaleway ARM nodes:
```

**Review by jcirinosclwy — APPROVED** [review]

**Pull request merged**